data-package/README.md

```diff
@@ -3,7 +3,7 @@ title: Data Package
 version: 1.0 
 author: Paul Walsh, Rufus Pollock
 created: 12 November 2007
-updated: 2 May 2017
+updated: 14 May 2020
```

Comment:
This shouldn't change - the substance of the spec has not changed 😉 

/cc @sglavoie

Reply:
Of course! I overlooked this important detail. :neutral_face: :+1: